packages/edit-site/src/components/global-styles/block-preview-panel.js

```diff
@@ -3,14 +3,9 @@
  */
 import { BlockPreview } from '@wordpress/block-editor';
 import { getBlockType, getBlockFromExample } from '@wordpress/blocks';
-import { useResizeObserver } from '@wordpress/compose';
 import { __experimentalSpacer as Spacer } from '@wordpress/components';
 
 const BlockPreviewPanel = ( { name, variation = '' } ) => {
-	const [
```

Comment:
Hm.. why we needed `useResizeObserver` in the first place here?

Reply:
To set the min-height of the preview iframe container, so that block preview (like button) is vertically center aligned.
It was captured using `useResizeObserver`, but seems like it was a bad idea for non-floating scrollbars that change with and height when they appear on content overflow.